src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -259,6 +264,7 @@ public string PathPrefix
         /// </summary>
         public bool UseSuccessStatusCodeRange { get; set; } = false;
 
+
         internal OpenApiConvertSettings Clone()
         {
             var newSettings = new OpenApiConvertSettings
```

Comment:
clone should contain the configuration value

Reply:
Sure